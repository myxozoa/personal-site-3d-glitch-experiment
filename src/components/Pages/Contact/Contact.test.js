import React from 'react';
import { shallow } from 'enzyme';

import Component from './index';

describe('<Contact />', () => {
  it('renders correctly', () => {
    const component = shallow(<Component />);

    expect(component).toMatchSnapshot();
  });
});