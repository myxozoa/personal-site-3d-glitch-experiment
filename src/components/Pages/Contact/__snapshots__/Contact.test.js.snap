// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Contact /> renders correctly 1`] = `
<div
  className="container"
>
  <h2
    className="glitch"
    data-text="contact"
  >
    contact
  </h2>
  <div
    className="box"
  >
    <div>
      ronnie miksch
    </div>
    <div>
      software developer
    </div>
    <div>
      sf bay area
    </div>
    <div>
      510-356-7749
    </div>
    <div>
      mikschronnie@gmail.com
    </div>
    <div>
      https://github.com/myxozoa
    </div>
  </div>
</div>
`;
